frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -1,6 +1,6 @@
 # Contributing Data
 
-Cellxgene supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
+CellxGene supports a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you have single cell data that meet our [requirements](#dataset-requirements), please reach out to our curation team at [cellxgene@chanzuckerberg.com](mailto:cellxgene@chanzuckerberg.com).
```

Comment:
Are we allowed to used cellxgene without the CZ? and then wouldn't CellxGene be CELLxGENE?

https://docs.google.com/document/d/1wj2Eq7nmz8YugGTr_tAb-u3xxK6gk59fh0sFcquJ6oA/edit#bookmark=kix.p4ernyqhiaj3

Reply:
this is a general comment for all instances